pkg/fqdn/proxy/ipfamily/ipfamily.go

```diff
@@ -3,33 +3,41 @@
 
 package ipfamily
 
+import "golang.org/x/sys/unix"
+
 type IPFamily struct {
-	Name        string
-	IPv4Enabled bool
-	IPv6Enabled bool
-	UDPAddress  string
-	TCPAddress  string
-	Localhost   string
+	Name       string
+	UDPAddress string
+	TCPAddress string
+	Localhost  string
+
+	SocketOptsFamily              int
+	SocketOptsTransparent         int
+	SocketOptsRecvOrIgnoreDstAddr int
```

Comment:
This should be `SocketOptsRecvOrigDstAddr` (receive original destination address)

Reply:
the "power" of autocompletion :joy_cat: addressed and fixed - thanks!